.github/workflows/code-styling.yml

```
@@ -0,0 +1,72 @@
+name: Code styling
+on:
+  #pull_request:
+  #  branches:
+  #    - main
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 15 1/3 *" # 15th of every 3rd month, for now
```

Running this on a schedule seems like overkill for the moment, let's comment this out, then we can at least try it as a manual trigger (for #623)
```suggestion
 #  schedule:
 #    - cron: "0 0 15 1/3 *" # 15th of every 3rd month, for now
```